src/txverificationqueue.cpp

```diff
@@ -0,0 +1,52 @@
+#include "primitives/transaction.h"
+//#include "txverificationqueue.h"
+#include "main.h"
+//#include "coin.h"
+
+void CTxVerificationQueueEntry::setTX(const CTransaction& newTx)
+{
+    tx = newTx;
+};
+
+CTransaction& CTxVerificationQueueEntry::getTX()
```

Comment:
accessor method could return a const ref and could be const themselves:
const CTransaction& CTxVerificationQueueEntry::getTX() const
NodeId CTxVerificationQueueEntry::getNodeId() const

Reply:
done
https://github.com/andreysobol/zen/commit/0493d81b4ddfad1832426ac1dcfe2630ee138fcb